### ISSUE TITLE
It's not even running.

### ISSUE THREAD

**Issue by ivashchenko** [zsajjad/BusinessCard#4]
Windows 10 platform.

>yarn
>react-native run-android

It's compiling correctly, but bundle is not:

Metro Bundler ready.

Loading dependency graph, done.
error: bundling failed: Error: Unable to resolve module `AccessibilityInfo` from `G:\Projects\BusinessCard\node_modules\react-native\Libraries\react-native\react-native-implementation.js`: Module `AccessibilityInfo` does not exist in the Haste module map
 
It is crashing. Please, fix this issue.

**Comment by Alexey909** [comment]
hello. I didn't see your comment. Is it still not runnung?
